docs/guides/basics.md

```diff
@@ -128,6 +128,9 @@ public class HelloWorld {
     }
 }
 ```
+```
+```
```

Comment:
Not necessary?

Reply:
whoops, that's sloppy of me. I backed that out.